pages/linux/hexdump.md

```diff
@@ -0,0 +1,24 @@
+# hexdump
+
+> Display file contents in hexadecimal, decimal, octal, or ascii.
+> More information: <https://man7.org/linux/man-pages/man1/hexdump.1.html>.
+
+- Display the input offset in hexadecimal, followed by three columns of data in octal, per line:
+
+`hexdump -b {{path/to/file}}`
+
+- Display the input offset in hexadecimal, followed by same sixteen bytes in `%_p` format enclosed in `|` characters:
+
+`hexdump --canonical {{path/to/file}}`
+
+- Interpret only specified length bytes of the input:
+
+`hexdump --length {{7}} {{path/to/file}}`
```

Comment:
```suggestion
- Interpret only specified length bytes of the input:

`hexdump --length {{1..infinity}} {{path/to/file}}`
```

Optional fix: [range syntax can be used](https://github.com/tldr-pages/tldr/pull/9646).
Note: I am not sure whether length can't be 0.